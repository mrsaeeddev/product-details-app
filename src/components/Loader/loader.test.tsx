import React from 'react';
import renderer from 'react-test-renderer';

import Loader from './';

describe('Loader', () => {
    it('should render correctly using Snapshot', () => {
        const tree = renderer.create(<Loader />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('has 1 child', () => {
        const tree: any = renderer.create(<Loader />).toJSON();
        expect(tree.children.length).toBe(1);
    });
});