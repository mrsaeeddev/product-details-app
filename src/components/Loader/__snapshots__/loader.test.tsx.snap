// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly using Snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "justifyContent": "center",
      },
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
